File: Lightened Dream/SupportForm.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Linq;
using System.Windows.Forms;
namespace LightenedDream
{
	public partial class SupportForm
	{

		private void tmrLoad_Tick(System.Object sender, System.EventArgs e)
		{
			tmrLoad.Enabled = false;
			cmbType.Text = "Comment";
			try {
				txtOS.Text = Environment.OSVersion.ToString();

			} catch (Exception ex) {
			}
			txtReplyAddress.Focus();
			Application.DoEvents();
			webSupport.Url = new Uri("http://www.lucid-code.com/LightenedDream/Support");
		}

		private void btnCancel_Click(System.Object sender, System.EventArgs e)
		{
			Close();
		}

		private void btnOk_Click(System.Object sender, System.EventArgs e)
		{
			try {
				EmailService.EmailService objEmailService = new EmailService.EmailService();
				string strResult = objEmailService.Send(txtReplyAddress.Text, cmbType.Text, txtOS.Text, txtMessage.Text);

				if (strResult == "Success!") {
					MessageBox.Show("The message has been sent. Thank you for your " + cmbType.Text.ToLower() + "!", "Lightened Dream", MessageBoxButtons.OK, MessageBoxIcon.Information);
				} else {
					MessageBox.Show(strResult, "LightenedDream.Support.Send()", MessageBoxButtons.OK, MessageBoxIcon.Error);
				}

				Close();
			} catch (Exception ex) {
				MessageBox.Show(ex.Message, "LightenedDream.Support.Send()", MessageBoxButtons.OK, MessageBoxIcon.Error);
			}
		}
		public SupportForm()
		{
			InitializeComponent();
		}
	}
}

File: Lightened Dream/SupportForm.vb
Public Class SupportForm

  Private Sub tmrLoad_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles tmrLoad.Tick
    tmrLoad.Enabled = False
    cmbType.Text = "Comment"
    Try
      txtOS.Text = Environment.OSVersion.ToString
    Catch ex As Exception

    End Try
    txtReplyAddress.Focus()
    Application.DoEvents()
    webSupport.Url = New Uri("https://www.lucid-code.com/LightenedDream/Support")
  End Sub

    Private Sub btnCancel_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnCancel.Click
        Close()
    End Sub

    Private Sub btnOk_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnOk.Click
        Try
            Dim objEmailService As New EmailService.EmailService()
            Dim strResult As String = objEmailService.Send(txtReplyAddress.Text, cmbType.Text, txtOS.Text, txtMessage.Text)

            If strResult = "Success!" Then
                MessageBox.Show("The message has been sent. Thank you for your " & cmbType.Text.ToLower & "!", "Lightened Dream", MessageBoxButtons.OK, MessageBoxIcon.Information)
            Else
                MessageBox.Show(strResult, "LightenedDream.Support.Send()", MessageBoxButtons.OK, MessageBoxIcon.Error)
            End If

            Close()
        Catch ex As Exception
            MessageBox.Show(ex.Message, "LightenedDream.Support.Send()", MessageBoxButtons.OK, MessageBoxIcon.Error)
        End Try
    End Sub
End Class